pkg/collector/prometheus_collector.go

```diff
@@ -218,7 +218,7 @@ func (p *PrometheusCollector) newNodeMetrics() {
 	nodeInfo := prometheus.NewDesc(
 		prometheus.BuildFQName(namespace, "node", "nodeInfo"),
 		"Labeled node information",
-		[]string{"cpu_architecture"}, nil,
+		[]string{"cpu_architecture", "instance"}, nil,
```

Comment:
`instance` is always in prometheus output, why do we need it again?

```json
{
  "__name__": "kepler_pod_energy_stat",
  "block_devices_used": "0",
  "command": "admission-",
  "container": "kepler-exporter",
  "container_name": "admission-controller",
  "curr_bytes_read": "0",
  "curr_bytes_writes": "0",
  "curr_cache_miss": "45524",
  "curr_container_cpu_usage_seconds_total": "0",
  "curr_container_memory_working_set_bytes": "0",
  "curr_cpu_cycles": "5255485",
  "curr_cpu_instr": "2328511",
  "curr_cpu_time": "0",
  "endpoint": "http",
  "instance": "xxxxx",  <------ the instance is the node
  "job": "kepler-exporter",
  "namespace": "kepler",
  "pod": "kepler-exporter-5rvr2",
  "pod_name": "vpa-admission-controller-64cb9cfb4b-tlmfm",
  "pod_namespace": "kube-system",
  "service": "kepler-exporter",
  "total_bytes_read": "0",
  "total_bytes_writes": "0",
  "total_cache_miss": "2936117602",
  "total_container_cpu_usage_seconds_total": "1462",
  "total_container_memory_working_set_bytes": "31195136",
  "total_cpu_cycles": "524240944252",
  "total_cpu_instr": "273766542755",
  "total_cpu_time": "1243120140"
}
```

Reply:
https://github.com/sustainable-computing-io/kepler/blob/main/pkg/collector/prometheus_collector.go#L277 @rootfs , I suppose I don't need test again...